tools/SetupFlow/DevHome.SetupFlow/TaskGroups/AppManagementTaskGroup.cs

```diff
@@ -39,11 +39,9 @@ public class AppManagementTaskGroup : ISetupTaskGroup
 
     public void HandleSearchQuery(string query)
     {
-        var searchParameter = HttpUtility.ParseQueryString(query)["search"];
-        if (!string.IsNullOrEmpty(searchParameter))
+        if (!string.IsNullOrEmpty(query))
         {
-            var trimmedSearchParameter = searchParameter.Trim('\"');
-            _appManagementViewModel.PerformSearch(trimmedSearchParameter);
+            _appManagementViewModel.PerformSearch(query);

```

Comment:
This call resolves into SearchViewModel.SearchAsync.  `SearchAsync` handles a null or whitepsace string.  No need to check here.